### PR TITLE
#2071 Deprecated polarsys links updated with Eclipse

### LINKED ISSUE #2071: Capella Welcome page reference deprecated polarsys links, to update towards Eclipse
Fix 3 links in Welcome page of all products (Capella and Melody/Team):

- [ ] Visit Capella Website

- [ ] Visit Capella Wiki

- [ ] Visit Capella forum
- [ ]  fix similar references in Thales wiki, Twiki (Orchestra & MSBE domains)
- [ ] same in Capella pages in Eclipse domain (deprecated links)

### PR DESCRIPTION
**Pull request by MalinaStefaniaStoicanescu**
Fixes #2071 

Change-Id: I91d30f4c4d7a2ecb08f1cd563371104237272b17
Signed-off-by: MalinaStefaniaStoicanescu <malina.stoicanescu@thalesgroup.com>

### CODE REVIEW

**Review by georgiana-ecobici — APPROVED** [review]

File: doc/plugins/org.polarsys.capella.developer.doc/html/Tutorials.mediawiki [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 Some online tutorials are available to ease creation of Capella Extensions.
 
-Please visit the online documentation on polarsys.org : https://wiki.polarsys.org/Capella/Tutorials
+Please visit the online documentation on polarsys.org : https://wiki.eclipse.org/Capella/Tutorials
```
Comment (pdulth) [inline doc/plugins/org.polarsys.capella.developer.doc/html/Tutorials.mediawiki:5]: https://github.com/eclipse/capella/wiki/Tutorials

File: doc/plugins/org.polarsys.capella.doc/html/Release note/KnownIssues.mediawiki [hunk 2]
```diff
@@ -1,3 +1,3 @@
 = Known Issues =
 
-[https://bugs.polarsys.org/buglist.cgi?bug_status=UNCONFIRMED&bug_status=CONFIRMED&order=bug_id%20DESC&product=Capella&query_based_on=&query_format=advanced&resolution=--- Known issues]
+[https://bugs.eclipse.org/bugs/=--- Known issues]
```
Comment (pdulth) [inline doc/plugins/org.polarsys.capella.doc/html/Release note/KnownIssues.mediawiki:3]: https://github.com/eclipse/capella/wiki/Repositories

File: doc/plugins/org.polarsys.capella.core.ui.intro/intro/firststeps-doc.xml [hunk 3]
```diff
@@ -18,15 +18,15 @@
...
                 <text>Read Case-Studies, attend to Upcoming Events, access to Training and Coaching, etc.</text>
             </link>
             <link style-id="content-link" label="Visit Capella Wiki"
-                url="http://org.eclipse.ui.intro/openBrowser?url=https://wiki.polarsys.org/Capella" id="visitCapellaWiki">
+                url="http://org.eclipse.ui.intro/openBrowser?url=https://wiki.eclipse.org/Capella" id="visitCapellaWiki">
                 <text>Watch "how-to" and demonstration screencasts, learn more about the next evolutions and roadmaps, access to development resources, etc.</text>
             </link>
             <link style-id="content-link" label="Visit Capella Forum"
-                url="http://org.eclipse.ui.intro/openBrowser?url=https://polarsys.org/forums/index.php" id="visitCapellaForum">
+                url="http://org.eclipse.ui.intro/openBrowser?url=https://www.eclipse.org/forums/" id="visitCapellaForum">
```
Comment (pdulth) [inline doc/plugins/org.polarsys.capella.core.ui.intro/intro/firststeps-doc.xml:29]: forum.mbse-capella.org/

**Review by pdulth — CHANGES_REQUESTED** [review]

File: doc/plugins/org.polarsys.capella.doc/html/Release note/KnownIssues.mediawiki (hunk 2 above)
Comment (MalinaStefaniaStoicanescu) [reply doc/plugins/org.polarsys.capella.doc/html/Release note/KnownIssues.mediawiki]: done

File: doc/plugins/org.polarsys.capella.core.ui.intro/intro/firststeps-doc.xml (hunk 3 above)
Comment (MalinaStefaniaStoicanescu) [reply doc/plugins/org.polarsys.capella.core.ui.intro/intro/firststeps-doc.xml]: done

File: doc/plugins/org.polarsys.capella.developer.doc/html/Tutorials.mediawiki (hunk 1 above)
Comment (MalinaStefaniaStoicanescu) [reply doc/plugins/org.polarsys.capella.developer.doc/html/Tutorials.mediawiki]: done

File: doc/plugins/org.polarsys.capella.developer.doc/html/Tutorials.mediawiki [hunk 4]
```diff
@@ -2,4 +2,4 @@
 
 Some online tutorials are available to ease creation of Capella Extensions.
 
-Please visit the online documentation on polarsys.org : https://wiki.polarsys.org/Capella/Tutorials
+Please visit the online documentation on polarsys.org : https://github.com/eclipse/capella/wiki/Tutorials
```
Comment (pdulth) [inline doc/plugins/org.polarsys.capella.developer.doc/html/Tutorials.mediawiki:5]: can you remove also "on polarsys.org ", as this site no longer exist eighter

File: doc/plugins/org.polarsys.capella.core.ui.intro/intro/firststeps-doc.xml [hunk 5]
```diff
@@ -18,15 +18,15 @@
         path="firststeps/@">
         <group id="content-group" style-id="content-group">
             <link style-id="content-link" label="Visit Capella Website"
-                url="http://org.eclipse.ui.intro/openBrowser?url=http://www.polarsys.org/capella/index.html" id="visitCapellaWebsite">
+                url="http://org.eclipse.ui.intro/openBrowser?url=https://www.eclipse.org/capella/index.html" id="visitCapellaWebsite">
                 <text>Read Case-Studies, attend to Upcoming Events, access to Training and Coaching, etc.</text>
             </link>
             <link style-id="content-link" label="Visit Capella Wiki"
-                url="http://org.eclipse.ui.intro/openBrowser?url=https://wiki.polarsys.org/Capella" id="visitCapellaWiki">
+                url="http://org.eclipse.ui.intro/openBrowser?url=https://wiki.eclipse.org/Capella" id="visitCapellaWiki">
```
Comment (pdulth) [inline doc/plugins/org.polarsys.capella.core.ui.intro/intro/firststeps-doc.xml:25]: Ups this one doesn't work either,  its https://github.com/eclipse/capella/wiki

File: doc/plugins/org.polarsys.capella.developer.doc/html/Tutorials.mediawiki (hunk 4 above)
Comment (MalinaStefaniaStoicanescu) [reply doc/plugins/org.polarsys.capella.developer.doc/html/Tutorials.mediawiki]: done

File: doc/plugins/org.polarsys.capella.core.ui.intro/intro/firststeps-doc.xml (hunk 5 above)
Comment (MalinaStefaniaStoicanescu) [reply doc/plugins/org.polarsys.capella.core.ui.intro/intro/firststeps-doc.xml]: done

**Review by pdulth — APPROVED** [review]

**Pull request merged**